cypress/integration/README.md

```diff
@@ -22,6 +24,12 @@ Key points:
    - testsForALLAMPPages.js or testsForAllCanonicalPages.js
    - /[pageName]/tests.js
    - /[pageName]/testsForAMPOnly.js or /[pageName]/testsForCanonicalOnly.js
+ - Inside each of the files in the list above there are two sets of tests:
+   - testsToAlwaysRun (as the name implies these will always run, use this sparingly because it will run for every service!)
+     - as such tests bypass smoke test rules logic, limiting the services/pages they run on with conditonal logic is strongly encouraged - these tests are deliberately not given their own config for service:pageType combinations to run on to keep config maintainable and to reduce the cognitive complexity of these tests rather than increase the level of abstraction. This may cause a wasted page visit during testing.
```

Comment:
```suggestion
     - We recommend using conditional logic to select just the services/pagetypes that are necessary. e.g. only run ATI analytics tests on `afaanoromoo, cymrufyw, japanese, naidheachdan, news` instead of all 44 services, since these cover all variants. 
```